idea/src/META-INF/gradle-java.xml.as33

```diff
@@ -78,6 +78,7 @@
 
         <scriptDefinitionContributor implementation="org.jetbrains.kotlin.idea.core.script.GradleScriptDefinitionsContributor" order="first"/>
 
+        <moduleBuilder implementation="org.jetbrains.kotlin.ide.konan.gradle.KotlinGradleNativeMultiplatformModuleBuilder"/>
```

Comment:
You should (probably) change also `gradle.xml.181`, otherwise your wizard will not bee seen in IDEA 181 or earlier versions.

Reply:
OK.